app/Http/Controllers/Auth/LoginController.php

```diff
@@ -55,4 +56,14 @@ public function showLoginForm()
 
         return view('auth.login');
     }
+
+    protected function loggedOut(Request $request)
+    {
+        $custom_logout_handler = Config::get('auth_logout_handler');
```

Comment:
Seems like one line: 
```php
return redirect(Config::get('auth_redirect_handler', $this-redirectTo));
```

Reply:
Agreed, didn't realize that default values were supported by Config::get.